docs/user-guide.md

```diff
@@ -16,7 +32,154 @@
 
 ### Settings
 
-To ensure that the Liberty Config Language Server starts properly, we advise users define the `xml.java.home` property in their VS Code settings.
+Liberty Tools for Visual Studio Code requires Java 17 for functionality.
```

Comment:
java 17 is listed as a "Software requirement" in Intellij- would it not be the same here?  requirements